### DISCUSSION TITLE
Not much documentation for LEDOutput

### DISCUSSION THREAD

**Discussion by AyceBenz** [pschatzmann/arduino-audio-tools#1795] [Q&A] [answered]
### Problem Description

NOT a bug! but cant find any options to post an issue

used the doxygen thing for reference
still could NOT get ledoutput to work
forgot to save the sketch

### Device Description

ESP32-WROOM-32 

### Sketch

```cpp
.
```

### What is your development environment

Arduino

### I have checked existing issues, discussions and online documentation

- [X] I confirm I have checked existing issues, discussions and online documentation

**Comment by pschatzmann** [comment]
So, why don't you just create a discussion ?

**Comment by pschatzmann** [comment, marked as answer]
There are different scenarios: one of them is to use it with FFT like in [this example](https://github.com/pschatzmann/arduino-audio-tools/blob/2d5d09aeb6e2c0edb700dd3c786062c5802732d1/examples/examples-communication/a2dp/basic-a2dp-fft-led/basic-a2dp-fft-led.ino#L11) or [this](https://github.com/pschatzmann/arduino-audio-tools/blob/2d5d09aeb6e2c0edb700dd3c786062c5802732d1/examples/examples-audiokit/streams-audiokit-fft-led/streams-audiokit-fft-led.ino#L15)

**Reply by AyceBenz** [reply]
Thanks! Will look into this

**Reply by AyceBenz** [reply]
sorry about the issue thing i dont use github much really
